README.md

```diff
@@ -72,11 +72,11 @@ If you want to **update a portable version**, you should delete all the files an
 <table>
   <tr>
     <td>
-      <strong>Latest Release: v4.2.1</strong>
+      <strong>Latest Release: v5.0</strong>
```

Comment:
The test error has `FormUpdates.cs` in the callstack.
Might it work with
```suggestion
      <strong>Latest Release: v5.0.0</strong>
```
?

On the other hand, we disable update checks for test runs IIRC (cannot check right now).

Reply:
I forgot to bump the version in the appveyor.yml file, and that triggers the Update dialog to appear.